volatility3/framework/symbols/windows/extensions/registry.py

```diff
@@ -159,14 +159,22 @@ class CM_KEY_NODE(objects.StructType):
     """Extension to allow traversal of registry keys."""
 
     def get_volatile(self) -> bool:
+        """
+        Returns a bool indicating whether or not the key is volatile.
+
+        Raises ValueError if the key was not instantiated on a RegistryHive layer
```

Comment:
Unfortunately this raises `ValueError` when the other methods raise `TypeError` for the same reason, but I left it as is to avoid the major version bump that I suspect would be required if we changed this.